pyuvsim/uvsim.py

```diff
@@ -98,6 +98,14 @@ def __init__(self, sources, time, freq, baseline, telescope, freq_i=0):
         if sources.spectral_type == 'flat':
             self.freq_i = 0
 
+    def __repr__(self) -> str:
+        """Make a nice representation."""
+        return (
+            f"UVTask, [time: {self.time}, freq: {self.freq}, source names: "
```

Comment:
My only thought here is do we want `UVTask[ .....]` without the ',' or with '<' or '('

Reply:
Yes, I think we should ditch the space and comma. I don't have a preference for '[' vs '<' vs '('. Do you?

Reply:
I don't have a preference myself.

Reply:
done! I went with '<'.